src/containers/ListingPage/SectionHeading.js

```diff
@@ -27,6 +27,7 @@ const SectionHeading = props => {
     ? 'ListingPage.perDay'
     : 'ListingPage.perUnit';
 
+  const hasCertificate = certificate === '200h' || certificate === '500h';
```

Comment:
I'd make this the other way around. People might add more certificates, so checking against "no certificate" is less error-prone.